Content.Server/RatKing/RatKingSystem.cs

```diff
@@ -130,10 +133,10 @@ private void OnPointedNearby(EntityUid uid, RatKingComponent component, ref Afte
                     continue;
 
                 // Remove ineligable targets
-                if (!TryComp<DamageableComponent>(ent, out var _1) || !TryComp<BodyComponent>(ent, out var _2))
+                if (!HasComp<DamageableComponent>(ent) || !HasComp<BodyComponent>(ent) || !_container.IsEntityInContainer(ent))
```

Comment:
```suggestion
                if (!HasComp<DamageableComponent>(ent) || !HasComp<BodyComponent>(ent) || _container.IsEntityInContainer(ent))
```

is that `!` supposed to be there? I think you want to exlude it if the entity *is* in a container?